### PR TITLE
Passport

### PR DESCRIPTION
**Pull request by computergeek12**
- Forgot to add app.use()
- Added the callback

### CODE REVIEW

**Comment by CreeperPlanet26** [comment]
Please refrain from using var and use modern es6 features such as let or const

**Comment by CreeperPlanet26** [comment]
also you have declared passport twice which breaks the code

**Comment by computergeek12** [comment]
Oops, my bad.

**Comment by computergeek12** [comment]
Also I didn't declare it twice, is it because it tried to merge my file with the other file?

**Comment by computergeek12** [comment]
I changed all instances of var to let. I personally like var better, but fine.

**Comment by CreeperPlanet26** [comment]
also some things that you don't change should be const

File: server/src/app.js [hunk 1]
```diff
@@ -2,10 +2,73 @@
...
 
+const Guild = require("./database/models/Guild");
+const User = require("./database/models/User");
+
+const passport = require("passport");
+const { Strategy } = require("passport-discord");
+
+const { Client, Permissions } = require('discord.js');
+const client = new Client();
+client.login(process.env.BOT_TOKEN);
```
Comment (CreeperPlanet26) [inline server/src/app.js:13]: where is discord.js coming from? this is the backend there is no discord.js

Comment (computergeek12) [reply server/src/app.js:13]: Oh well... I need discord.js to check mutual servers and to convert the permissions integer. This way, I'd need another method...

Comment (CreeperPlanet26) [reply server/src/app.js:13]: You have to fetch from the discord api like it says in the read me.

**Comment by computergeek12** [comment]
Sure.

**Comment by computergeek12** [comment]
Removed discord.js and directly referenced the API.

**Comment by fracturx** [comment]
Please pull the remote changes in and resolve any merge conflicts that arise

**Comment by computergeek12** [comment]
Alright, the simplest thing I'm going to do is close this PR, make a DiscordStrategy file and merge conflicts, then make a new one.

**Comment by fracturx** [comment]
👍

**Pull request closed without merging**